packages/editor/src/components/hierarchy/HierarchyPanelContainer.tsx

```diff
@@ -483,11 +488,40 @@ function HierarchyPanelContents({ rootEntityUUID }: { rootEntityUUID: EntityUUID
             fontSize: '12px',
             lineHeight: '0.5'
           }}
-          onClick={() => EditorControlFunctions.createObjectFromSceneElement()}
+          onClick={(event) => {
+            anchorElement.set(event.currentTarget)
+            if (getMutableState(EditorHelperState).studioMode.value == 'Advanced') {
+              EditorControlFunctions.createObjectFromSceneElement()
+            }
+          }}
         >
           {t('editor:hierarchy.lbl-addEntity')}
```

Comment:
we should change the language here for Add Entity in simple mode to Add Object

Reply:
This lines mean if user select Advanced mode the studio will be same.
When select simple mode the fuctions of add object are in PrefabList.tsx which was called in HierarchyPanelContainer.tsx line 523.